pkg/store/types/matcher.go

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package storetypes
```

Comment:
Empty file? :smile:

Reply:
Yeah, was refactoring stuff around and forgot to delete.